src/module/item/mixins/item-activation.js

```diff
@@ -108,17 +106,6 @@ export const ItemActivationMixin = (superclass) => class extends superclass {
 
                     if (!active) chatData.action = "SFRPG.ChatCard.ItemActivation.Deactivates";
 
-                    // Toggle default roll mode
-                    if (["gmroll", "blindroll"].includes(rollMode)) {
-                        chatData["whisper"] = ChatMessage.getWhisperRecipients("GM");
-                    }
-                    if (rollMode === "blindroll") {
-                        chatData["blind"] = true;
-                    }
-                    if (rollMode === "selfroll") {
-                        chatData["whisper"] = ChatMessage.getWhisperRecipients(game.user.name);
-                    }
-
                     ChatMessage.create(chatData, { displaySheet: false });
```

Comment:
After some digging, it looks like this whole section can be replaced with if we want to keep the private text functionality
```js
ChatMessage.applyRollMode(chatData, rollMode);`
ChatMessage.create(chatData, { displaySheet: false, rollMode: rollMode });
```